packages/react/src/components/Toolbar/ToolbarMenuItem.tsx

```diff
@@ -115,6 +154,24 @@ class ToolbarMenuItem extends UIComponent<WithAsProp<ToolbarMenuItemProps>> {
       </ElementType>
     )
 
+    const hasChildren = childrenExist(children)
+
+    if (popup && !hasChildren) {
+      return Popup.create(popup, {
+        defaultProps: {
+          trapFocus: true,
```

Comment:
I agree that this is more convenient, but since trapFocus is not the default for normal <Popup /> we probably should not put it here and be consistent.

Reply:
`ToolbarItem` also sets `trapFocus: true`

https://github.com/stardust-ui/react/blob/master/packages/react/src/components/Toolbar/ToolbarItem.tsx#L225

Reply:
didn't know about it. ok.